### DISCUSSION TITLE
micromark-extension-gfm seems to break Blockquote laziness rule

### DISCUSSION THREAD

**Discussion by eliyap** [micromark/micromark#69] [Q&A] [answered]
Hi! 
I'm running into unexpected behavior with MDAST:

**Setup**
```javascript
var gfm = require("mdast-util-gfm");
var fromMarkdown = require("mdast-util-from-markdown");
var syntax = require("micromark-extension-gfm");

var mdast = fromMarkdown(markdown, {
    extensions: [syntax()],
    mdastExtensions: [gfm.fromMarkdown]
});

let mdast = fromMarkdown(`> Quote  <-- 2 spaces
Quote Continuation`);
```

**Expected**
MDAST with a blockquote containing 2 paragraphs (Quote, Quote Continuation)
I get this outcome if I remove `syntax`, that is, I disable `micromark-extension-gfm`.

GFM allows for [laziness in blockquotes](https://github.github.com/gfm/#example-210).

**Actual**
MDAST with a blockquote (Quote) and a non-quoted paragraph (Quote Continuation)

Please could I check if this is user error or expected behavior? Thank you!

**Comment by ChristianMurphy** [comment, marked as answer]
It is not expected, it was an issue https://github.com/remarkjs/remark-gfm/issues/3
That said it should have been fixed in https://github.com/micromark/micromark/releases/tag/3.0.0, https://github.com/syntax-tree/mdast-util-from-markdown/releases/tag/1.0.0, and https://github.com/micromark/micromark-extension-gfm/releases/tag/1.0.0 .
What version(s) are you testing with?

**Reply by eliyap** [reply]
Ah, I hadn't updated my packages and just saw the 1.0.0 release. Will check my versions before filing next time. 

After upgrading, I observe the expected behaviour.

Thank you for the help!
